scripts/mitochondria_m2_wdl/MitochondriaPipeline.wdl

```diff
@@ -0,0 +1,558 @@
+  File shifted_ref_fasta_index
+  File shifted_ref_dict
+
+  Int? preemptible_tries
+  Int disk_size = ceil(size(input_bam_regular_ref, "GB") + size(input_bam_shifted_ref, "GB") + size(ref_fasta, "GB") * 2) + 20
+
+  meta {
+    description: "Remove this hack once there's a GVCF solution."
+  }
+  command <<<
+    set -e
+
+    java -jar /usr/gitc/picard.jar CollectHsMetrics \
+      I=${input_bam_regular_ref} \
+      R=${ref_fasta} \
+      PER_BASE_COVERAGE=non_control_region.tsv \
+      O=non_control_region.metrics \
+      TI=${non_control_region_interval_list} \
+      BI=${non_control_region_interval_list} \
+      COVMAX=20000 \
+      SAMPLE_SIZE=1
+
+    java -jar /usr/gitc/picard.jar CollectHsMetrics \
+      I=${input_bam_shifted_ref} \
+      R=${shifted_ref_fasta} \
+      PER_BASE_COVERAGE=control_region_shifted.tsv \
+      O=control_region_shifted.metrics \
+      TI=${control_region_shifted_reference_interval_list} \
+      BI=${control_region_shifted_reference_interval_list} \
+      COVMAX=20000 \
+      SAMPLE_SIZE=1
+
+    R --vanilla <<CODE
+      shift_back = function(x) {
+        if (x < 8570) {
+          return(x + 8000)
+        } else {
+          return (x - 8569)
+        }
+      }
+
+      control_region_shifted = read.table("control_region_shifted.tsv", header=T)
+      shifted_back = sapply(control_region_shifted[,"pos"], shift_back)
+      control_region_shifted[,"pos"] = shifted_back
+
+      beginning = subset(control_region_shifted, control_region_shifted[,'pos']<8000)
+      end = subset(control_region_shifted, control_region_shifted[,'pos']>8000)
+
+      non_control_region = read.table("non_control_region.tsv", header=T)
+      combined_table = rbind(beginning, non_control_region, end)
+      write.table(combined_table, "per_base_coverage.tsv", row.names=F, col.names=T, quote=F, sep="\t")
+
+    CODE
+  >>>
+  runtime {
+    disks: "local-disk " + disk_size + " HDD"
+    memory: "1200 MB"
+    docker: "us.gcr.io/broad-gotc-prod/genomes-in-the-cloud:2.4.1-1540490856"
+    preemptible: select_first([preemptible_tries, 5])
+  }
+  output {
+    File table = "per_base_coverage.tsv"
+  }
+}
+
+task SplitMultiallelics {
+  File input_vcf
+  File input_vcf_index
+  Int? preemptible_tries
+
+  File ref_fasta
+  File ref_fasta_index
+  File ref_dict
+
+  command <<<
+python <<CODE
+from subprocess import Popen,PIPE,check_output
+
+#split variants
+cmd = """bcftools norm -m -any ${input_vcf} > pre1_split.vcf""".format(**locals())
+result, err = Popen([cmd], stdout=PIPE, stderr=PIPE, shell=True).communicate()
+
+#edit Number
+cmd = """cat pre1_split.vcf|sed s'/Number=R/Number=1/g' > pre2_split.vcf""".format(**locals())
+result, err = Popen([cmd], stdout=PIPE, stderr=PIPE, shell=True).communicate()
+
+#left align the variants
+cmd = """bcftools norm -f ${ref_fasta} pre2_split.vcf  > split.vcf""".format(**locals())
+result, err = Popen([cmd], stdout=PIPE, stderr=PIPE, shell=True).communicate()
+CODE
```

Comment:
I should probably remove this from the gatk version, but unfortunately it has to be in the firecloud version for now, since it's for Kristen. I think the long term solution is that hail should be able to handle multi-allelics? I forget where we landed on this.